### ISSUE TITLE
Allow customizing `remarkPlugins` and `rehypePlugins`?

### ISSUE THREAD

**Issue by karlhorky** [nkzw-tech/remdx#25]
Hi @cpojer ! What are your thoughts on allowing customizing `remarkPlugins` and `rehypePlugins`?

https://github.com/nkzw-tech/remdx/blob/53058eb54c692c63885d7338086c9e48baad073a/packages/vite-plugin-remdx/index.ts#L169-L173

I was looking for this just now to add [`remark-gfm`](https://github.com/remarkjs/remark-gfm) for Markdown tables

**Comment by cpojer** [comment]
If it's just that one, might be worth adding as a default to ReMDX itself maybe?

**Comment by karlhorky** [comment]
Oh, yeah maybe `remark-gfm` would be a good default (I can open a PR for this, if someone else doesn't do that).

However, maybe there's a different remark / rehype plugin that a user would want - what do you think about also allowing configurability, in a separate PR?

**Comment by cpojer** [comment]
Yeah, for now let's just allow passing a custom config, and merge it into the options object here without adding new dependencies.

**Comment by karlhorky** [comment]
Another candidate for a default plugin would be the recma plugin [`recma-mdx-import-media`](https://github.com/ipikuka/recma-mdx-import-media) by @talatkuyuk, which would enable images, videos, etc using relative paths 🪄✨

```mdx
![alt](./image.png)

<img src="./image.png" alt="alt" />

![alt](../blog-assets/image.png)

<img src="../blog-assets/image.png" alt="alt" />
```

**Comment by cpojer** [comment]
Yup I'm fine with those as defaults. Send a PR.
